### DISCUSSION TITLE
Asus Rog Scar Strix g15 Keystone.

### DISCUSSION THREAD

**Discussion by Rom0067** [seerge/g-helper#517] [Q&A]
Hello,

I was wondering if there could be  a way to use ghelper to interact with my keystone in the future.  Since uninstalling AC I can no longer configure it.  Thanks!

(The keystone is the little nfc key that plugs into the side of rog strix models)

**Comment by seerge** [comment]
@Rom0067 honestly, i have no idea how that works :) but if you want to contribute - that’s always welcomed

**Reply by Rom0067** [reply]
Thank you for replying and for a tool that has helped so many!

I have zero knowledge of coding. I would love to help but do not know how.  

If you are asking how the keystone itself works from what I have gathered it is like an nfc smartcard that AC allowed you to set a few custom options when you plug it in/ unplug it. Like log out of windows when unplugged.

Either way I was only curious to see if you had any info on it.  Thanks again and I look forward to the future of GHelper!
